companion/src/translations/companion_de.ts

```diff
@@ -452,22 +452,22 @@ die kann von der Menüsprache abweichen</translation>
     <message>
         <location filename="../apppreferencesdialog.ui" line="801"/>
         <source>Use releases (stable)</source>
-        <translation type="unfinished"></translation>
+        <translation>Nur Releases (stable)</translation>
     </message>
     <message>
         <location filename="../apppreferencesdialog.ui" line="806"/>
         <source>Use releases and release candidates (testing)</source>
-        <translation type="unfinished"></translation>
+        <translation>Releases und Release Candidates (testing)</translation>
```

Comment:
Kandidaten?

Reply:
I thought about that, but I would say that Release, Release Candidates and Nightly Builds a common terms in german and shouldn't be translated. Not shure

Reply:
It still okay, was just wondering.